test/e2e/flowaggregator_test.go

```diff
@@ -792,13 +863,19 @@ func checkRecordsForToExternalFlows(t *testing.T, data *TestData, srcNodeName st
 			checkPodAndNodeData(t, record, srcPodName, srcNodeName, "", "", data.testNamespace)
 			checkFlowType(t, record, ipfixregistry.FlowTypeToExternal)
 			assert.NotContains(t, record, "octetDeltaCount: 0", "octetDeltaCount should be non-zero")
+			if checkEgress {
+				checkEgressInfo(t, record, egressName, egressIP)
+			}
 		}
 	}
 
 	clickHouseRecords := getClickHouseOutput(t, data, srcIP, dstIP, "", false, false)
 	for _, record := range clickHouseRecords {
 		checkPodAndNodeDataClickHouse(data, t, record, srcPodName, srcNodeName, "", "")
 		checkFlowTypeClickHouse(t, record, ipfixregistry.FlowTypeToExternal)
+		if checkEgress {
+			checkEgressInfoClickHouse(t, record, egressName, egressIP)
+		}
 		// Since the OVS userspace conntrack implementation doesn't maintain
 		// packet or byte counter statistics, skip the check for Kind clusters
```

Comment:
This is orthogonal to this PR, but we no longer use userspace OVS when running Antrea on Kind. Can you take an action item to follow-up on this and potentially remove this special case?

Reply:
Sure, I will open a separate PR for it.